src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/db/AvroHelper.java

```diff
@@ -192,8 +192,18 @@ private static Object getRowObject(ResultSetMetaData metaData, ResultSet row, in
           byte[] blob = row.getBytes(columnIndex);
           return blob == null ? null : ByteBuffer.wrap(blob);
         }
+      case Types.CHAR:
+        {
+          return trimTrailingSpaces(row.getString(columnIndex));
+        }
       default:
         return row.getObject(columnIndex);
     }
   }
+
+  private static final Pattern TRAILING_SPACES_REGEX = Pattern.compile("\\s++$");
```

Comment:
Constants should be defined at the beginning of the class.

Reply:
Done